src/main/java/Trapezoid.java

```diff
@@ -0,0 +1,41 @@
+public class Trapezoid extends Figure{
+
+    private long[] bases = new long[2];
+
+    private long[] sides = new long[2];
+
+    private long height;
```

Comment:
it make sense use double instead of long values, that will make calculations more precise